File: core-api/src/main/java/com/sequenceiq/cloudbreak/api/model/AmbariAddressJson.java
package com.sequenceiq.cloudbreak.api.model;


import com.sequenceiq.cloudbreak.doc.ModelDescriptions;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

@ApiModel("AmbariAddress")
public class AmbariAddressJson implements JsonEntity {

    @ApiModelProperty(value = ModelDescriptions.AMBARI_SERVER, required = true)
    private String ambariAddress;

    public String getAmbariAddress() {
        return ambariAddress;
    }

    public void setAmbariAddress(String ambariAddress) {
        this.ambariAddress = ambariAddress;
    }
}

File: core-api/src/main/java/com/sequenceiq/cloudbreak/api/model/AmbariAddressJson.kt
package com.sequenceiq.cloudbreak.api.model


import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty

@ApiModel("AmbariAddress")
class AmbariAddressJson : JsonEntity {

    @ApiModelProperty(required = true)
    var ambariAddress: String? = null
}
